### ISSUE TITLE
error on add new feed

### ISSUE THREAD

**Issue by gutierri** [jangernert/FeedReader#578]
When I try to insert a new entry, the feedreader closes.

Debug:
> ** (feedreader:22976): DEBUG: UpdateButton: update status
** (feedreader:22976): DEBUG: Ping: status 200
** (feedreader:22976): DEBUG: Ping successful
** (feedreader:22976): DEBUG: Table CachedActions is empty
** (feedreader:22976): DEBUG: CachedActionManager - executeActions: no actions to perform
** (feedreader:22976): DEBUG: Ping: https://duckduckgo.com/
** (feedreader:22976): DEBUG: FeedReader: setOnline
** (feedreader:22976): DEBUG: Ping: status 200
** (feedreader:22976): DEBUG: Ping successful
** (feedreader:22976): DEBUG: FeedReader: updateSyncProgress
NULL
An error occured (SIGSEGV)
   The reason is likely a null reference being used.
   Note: no file path and line numbers can be retrieved. Are you sure you added -g -X -rdynamic to valac command line?
    # 1  <unknown>   in 'feed_reader_feed_server_syncContent'
        at feedreader
    # 2  <unknown>   in 'clone'
        at /usr/lib/libc.so.6

**Comment by brendanlong** [comment]
Which version are you using, and which backend plugin (local, Feedly, TT-RSS, etc?)

**Comment by gutierri** [comment]
From source and backend local

**Comment by bil-elmoussaoui** [comment]
I had the same issue with local source too. Using the master branch, i didn't have the time to fix it locally yet

**Comment by jangernert** [comment]
Does it happen with every feed you add? Any detail we should know about? I just tried it with a random rss feed and it worked fine for me.

**Comment by bil-elmoussaoui** [comment]
i'm getting this issue whenever i try to sync the local RSS 
```shell
An error occured (SIGSEGV)
   The reason is likely a null reference being used.
   Note: no file path and line numbers can be retrieved. Are you sure you added -g -X -rdynamic to valac command line?

    #1  <unknown>   in 'atoi'
        at 
    #2  <unknown>   in 'feed_reader_feed_server_syncContent'
        at feedreader
    #3  <unknown>   in 'clone'
        at /usr/lib/libc.so.6
```

**Comment by jangernert** [comment]
@brendanlong fixed this with #580
